scripts/post-promote-tasks-utils.ts

```diff
@@ -77,3 +84,29 @@ export async function getBaseAmpVersion(
     }
   }
 }
+
+export async function getPullRequestDetails(
```

Comment:
Since this only returns merged PRs, how about renaming it `getMergedPullRequestDetails`?

Reply:
not necessarily just merged PRs